public/js/monitor.js

```diff
@@ -226,29 +226,53 @@ const postData = (url, data = {}) => {
   .catch(error => console.error(error));
 };
 
+function showAdditionalBreaches(){
+  document.getElementById("show-additional-breaches").classList.toggle("hide");
+  const additionalBreaches = document.getElementById("additional-breaches");
+  additionalBreaches.classList.toggle("show-breaches");
+  //setting height this way enables transition easing... setting the new height to "auto" 
+  if (additionalBreaches.classList.contains("show-breaches")) {
+    additionalBreaches.style.height = additionalBreaches.scrollHeight + "px";
+  }
+}
+
+const handleRadioButtons = function(form) {
+  const inputFields = form.querySelectorAll(".radio-button-group, .button");
+  inputFields[0].querySelector("input").checked = true;
```

Comment:
Do we need a choice to be pre-checked? It may lead users to over-select this choice.

Reply:
This was in accordance with a Photon radio-button stipulation that one should always be checked, but I wondered about over-selection/lazy-selection also.

Reply:
Let's remove it ... that may apply more/better in Firefox UI where we have a good idea of sensible defaults. Here we'd like to keep it open-ended.